byterun/caml/misc.h

```diff
@@ -124,7 +124,94 @@ CAMLnoreturn_end;
 CAMLextern char * caml_strdup(const char * s);
 CAMLextern char * caml_strconcat(int n, ...); /* n args of const char * type */
 
-/* <private> */
+/* Use macros for some system calls being called from OCaml itself.
+  These calls can be either traced for security reasons, or changed to
+  virtualize the program. */
+
+#ifndef CAML_WITH_CPLUGINS
+
```

Comment:
cplugins can be used for many purposes, one of them being to intercept/monitor calls to the file-system. However, they could also be used to dynamically extend the runtime, for example for memory profiling and process monitoring (à la `ocamlviz`). Even if some calls do not work under Windows does not mean that all these uses would not work.